addons/armaos/functions/fnc_terminal_addChar.sqf

```diff
@@ -0,0 +1,35 @@
+params ["_computer", "_inputChar"];
+
+private _terminal = _computer getVariable "AE3_terminal";
+
+private _terminalBuffer = _terminal get "AE3_terminalBuffer";
+private _terminalCursorPosition = _terminal get "AE3_terminalCursorPosition";
+private _terminalMaxColumns = _terminal get "AE3_terminalMaxColumns";
+
+hint format ["TERMINAL BUFFER: %1", _terminalBuffer];
```

Comment:
Debug msg

Reply:
removed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/bdeec0439c3a89ef1583c3b663196a37a6c51e0b